### DISCUSSION TITLE
Spotlight Scrolling

### DISCUSSION THREAD

**Discussion by sillypnk** [mantinedev/mantine#1232] [Q&A] [answered]
Hello! Mantine is amazing and cool!

This is my first time using mantine with remix and totally love it.

So anyways, is it possible to show a `scrollbar` on the spotlight provider if you have more than 5 categories? trying the `overflow: auto` with `maxHeight: x` on the style (or sx) doesn't work.

**Comment by rtivital** [comment, marked as answer]
You can add these properties using Styles API, I would not recommend doing that because it would be a bad UX. Spotlight is designed the same way as autocomplete – user is supposed to search for content and then pick one of the search results.

**Reply by gubi** [reply]
how to solve?
I have a long result list in an Autocomplete but only first five are displayed.
I also see in https://mantine.dev/core/autocomplete/?t=props but there's no way to solve

**Reply by rtivital** [reply]
https://github.com/mantinedev/mantine/discussions/1330

**Reply by pranaliMohite** [reply]
@rtivital 
I am using spotlight for quick search in my application. Here are quick questions:
It seems we have to assign all possible search results  to actions. Then when we query in spotlight it will query in actions. We dont need to submit again server request for new query.


I have set specific number to limit and able to see scrollbar for spotlight component but onScroll event is not working in that case.

                <SpotlightProvider
                    onScroll={()=>alert('scrolled')}
                    actions={searchData}
                    limit={17}
                    highlightQuery
                    searchIcon={<IconSearch size={18} />}
                    searchPlaceholder="Search..."
                    shortcut="mod + shift + 1"
                    nothingFoundMessage="Nothing found..."
                  >{props.children}</SpotlightProvider>
